### PR TITLE
Ruby on Rails: working with active records

### PR DESCRIPTION
**Pull request by mahmoud717**
in this project, we created a database for the mini reddit forum using active record, migrations, and validation.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi Team 👋

**Good work on this. 👍**

However, let's fix this:

- [x] You have your associations all wrong. Please refer to comments on your code for instructions on how to fix this.

- [x] Please change the username attribute on the user model to name attribute and add an email attribute to the model.

- [x] Also kindly remove the title attribute from the comments model.

Please fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  validates :username, presence: true, length: { minimum: 5, maximum: 15 }, uniqueness: true
+  has_many :posts
+  has_many :comments, through: :posts
```
Comment (mikenath223) [inline app/models/user.rb]: - [x] The user should have many comments but not through posts.

File: app/models/comment.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  has_one :user, through: :post
```
Comment (mikenath223) [inline app/models/comment.rb:4]: - [x] Comments should simply belong to user same as post.

Comment (Haywhizzz) [reply app/models/comment.rb:4]: We have added the user to comments

**Review by MarioDena — CHANGES_REQUESTED** [review]
Hello Team good work on your project and Implementing the requested changes. However there are still a few things to be Improved. 

# Changes Required 🔴 

- [x] Right now if the user follow your Getting Started instructions the user can't run your project for preview. Please add all necessary steps (EX: db:migrate, etc)

Cheers! 👍

**Review by rOluochKe — APPROVED** [review]
Hello Team
### You did well, you got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**